### DISCUSSION TITLE
Not randomizing

### DISCUSSION THREAD

**Discussion by blondmanny** [hopollo/OBS-Youtube-Player#4] [General]
Hi! Everything's working great, I love the work you've done, however I can't get the videos to play in random other. whether I set the value to true or false, or remove it all together, it just plays in the original order, do you have any idea why? thank you.

**Comment by hopollo** [comment]
Hello, thanks for your feedback.

Last time i tried it was working great but i know i always had massive issues with the randomize stuff so i'm gonna double-check and rewrite the function as soon as i can (really busy atm). 

Peace

**Comment by KyleMcCarley** [comment]
I was having the same issue until I made the playlist "Unlisted" instead of "Private."

**Reply by KyleMcCarley** [reply]
Issue's happening again, actually. Can't seem to fix it now :/

**Comment by marcelodeepspacepodcast** [comment]
Hello!
This issue is still on, maybe is because that function is called shuffle in youtube? If you change from 'random' to 'shuffle'?
